debian/changelog

```diff
@@ -1,3 +1,19 @@
+opx-nas-l3 (3.9.0) unstable; urgency=medium
+
+  * Update: Added UT script for PBR ACL cleanup dependency issue
+  * Bugfix: 'show ip arp summary' and 'ipv6 neighbor summary' do not clear the 
+            count on shutting down the ICL ports.
+  * Bugfix: arp mismatch is seen between vlt peers after a node reload on a TOR switch.
+  * Update: NHT support for non-default VRF.
+  * Update: Added support for virtual rif attribute as part of VRRP peer routing MAC configuration.
+  * Update: Updated NHT UT scripts to use the logs for validation based on test start time.
+  * Bugfix: VLAN LLA missing from "l3 ip6host show" table resulting in failure to learn IPv6 host entries via traffic
+  * Bugfix: Missing arp entries on vlt peer
```

Comment:
Remove mention of fixes having to do with VLT since they are not relevant to OPX by itself. Make sure to update commit message with the same changes.